README.md

```diff
@@ -27,9 +27,9 @@ Windows, Linux, FreeBSD, NetBSD and OpenBSD. Go to the [releases
 page](https://github.com/cortesi/modd/releases/latest), download the package for
 your OS, and copy the binary to somewhere on your PATH.
 
-If you have a working Go installation, you can also say
+Alternatively, with Go 1.17+ installed, you can install directly with CGO enabled.
 
-    $ go install github.com/cortesi/modd/cmd/modd@latest
+    $ CGO_ENABLED=1 go install github.com/cortesi/modd/cmd/modd@latest
```

Comment:
I don't really love using the `CGO_ENABLED` prefix here, as it suggest that this is abnormal, when in reality Go has CGO enabled by default, so not having it enabled is out of the ordinary.

Perhaps a better way of wording it might be:

```
Alternatively, with Go 1.17+ installed, you can install `modd` using the Go tooling.

    $ go install github.com/cortesi/modd/cmd/modd@latest

Please note that CGO is required, so if you happen to have it disabled you will need to add the `CGO_ENABLED=1` environment variable.

    $ CGO_ENABLED=1 go install github.com/cortesi/modd/cmd/modd@latest

Reply:
Updated based on comments. Thanks!